### ISSUE TITLE
Feature Request: Allow choosing where calendar is inserted 

### ISSUE THREAD

**Issue by lsharkey** [sawhney17/logseq-calendars-plugin#35]
This is such a great plugin!  Thanks for writing it.  My only pain point with it is that it inserts the cal events at the bottom of the page, and I would like them to be at the top.  Alternately, I would like to insert them to a different page, which I could keep open in the sidebar.  I hope you will consider adding this ability.  Thanks!

**Comment by sawhney17** [comment]
Interesting. Could you give some more  details about how it would work in insertion them on a mother page? Paritucllry how would the export format be.

**Comment by elfrinjo** [comment]
Would it be an option to insert the calendar block at a predefined position by replacing a keyword that could have come from a template?

**Comment by kennethaar** [comment]
Yeah top of page or at {calendar placeholder} would be very nice.

**Comment by jaker-dotcom** [comment]
There is the logseq quicktodo plugin, that allows insertion as children of a preconfigured block on the Journal page. Maye it's code can be helpful.
